src/java.base/share/classes/java/io/IO.java

```diff
@@ -35,6 +35,9 @@
  * is {@code null}; otherwise, the effect is as if a similarly-named method
  * had been called on that console.
  *
+ * <p> Output from methods in this class uses the character set of the system
```

Comment:
Hello Pavel, given that `java.io.IO` exposes both input and output methods and the Charset will be used by both input and output, I think we might have to reword this sentence to make a note of input too.